library/src/lib/toggle/toggle.component.ts

```diff
@@ -106,35 +103,37 @@ export class ToggleComponent implements OnInit, ControlValueAccessor {
     }
 
     /**
-     * @hidden 
+     * @hidden
      * @param value Sets the value of the *checked* property of the toggle.
      */
     writeValue(value: any) {
         this.checked = value;
     }
 
-    /** 
-     * @hidden 
```

Comment:
what are all these space removals for?

Reply:
I think my IDE's linter does this automatically